applications/prompt-keda-lsstcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -18,7 +18,7 @@ prompt-keda:
       main: |-
         - survey: BLOCK-365  # FBS SV Field Survey
           pipelines:
-          # TODO: enable ApPipe only when we're ready (resources, alerts, etc.)
+          - ${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/ApPipe.yaml
```

Comment:
What about all the other blocks?

Reply:
We're not supposed to need them, but I didn't  get a clear answer at https://rubin-obs.slack.com/archives/C07QM720NJG/p1745258179402279 about what happened.

Reply:
Maybe @ebellm  or @isullivan feels comfortable to say  that it's okay to ignore SV blocks that aren't properly accompanied with the expected survey name?

Reply:
Removed on #4614.